libraries/src/Installer/Adapter/TemplateAdapter.php

```diff
@@ -332,13 +334,17 @@ protected function parseQueries()
 					'0',
 					Text::sprintf('JLIB_INSTALLER_DEFAULT_STYLE', Text::_($this->extension->name)),
 					$this->extension->params,
+					intval($this->manifest->inheritable),
```

Comment:
```suggestion
					(int) $this->manifest->inheritable,
```
is there a reason why you not just cast to int?